downloads.html

```diff
@@ -4,329 +4,361 @@
 	<!--#include virtual="includes/_nav.htm" -->
 	<div class="right">
 		<h1>Download</h1>
-    <p>0.10.1.1 is the latest release. The current stable version is 0.10.1.1.</p>
+    <p>0.10.2.0 is the latest release. The current stable version is 0.10.2.0.</p>
 
     <p>
     You can verify your download by following these <a href="http://www.apache.org/info/verification.html">procedures</a> and using these <a href="http://kafka.apache.org/KEYS">KEYS</a>.
+    </p>
 
-    <h3>0.10.1.1</h3>
-    <ul>
-      <li>
-        <a href="https://archive.apache.org/dist/kafka/0.10.1.1/RELEASE_NOTES.html.">Release Notes</a>
-      </li>
-       <li>
-        Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka-0.10.1.1-src.tgz">kafka-0.10.1.1-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka-0.10.1.1-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka-0.10.1.1-src.tgz.md5">md5</a>)
-      </li>
-       <li>
-        Binary downloads:
+        <h3>0.10.2.0</h3>
         <ul>
-          <li>Scala 2.10 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka_2.10-0.10.1.1.tgz">kafka_2.10-0.10.1.1.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.10-0.10.1.1.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.10-0.10.1.1.tgz.md5">md5</a>)
-          </li>
-          <li>Scala 2.11 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka_2.11-0.10.1.1.tgz">kafka_2.11-0.10.1.1.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.11-0.10.1.1.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.11-0.10.1.1.tgz.md5">md5</a>)
-          </li>
-          <li>Scala 2.12 (pre-alpha) &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz">kafka_2.12-0.10.1.1.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz.md5">md5</a>)
-          </li>
+            <li>
+                <a href="https://archive.apache.org/dist/kafka/0.10.2.0/RELEASE_NOTES.html">Release Notes</a>
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka-0.10.2.0-src.tgz">kafka-0.10.2.0-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.2.0/kafka-0.10.2.0-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.2.0/kafka-0.10.2.0-src.tgz.md5">md5</a>)
```

Comment:
@guozhangwang Looks like the download link is slightly wrong -- it has 0.10.1.1 as directory and 0.10.2.0 in the link. I think it's just this one source download, the rest of the binary artifacts look ok.